<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trackVideoWidget</class>
 <widget class="QWidget" name="trackVideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ProjectorDragCorner" name="ulCorner" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="cornerIndex" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="ProjectorDragCorner" name="urCorner" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="cornerIndex" stdset="0">
    <number>3</number>
   </property>
  </widget>
  <widget class="ProjectorDragCorner" name="blCorner" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="cornerIndex" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="ProjectorDragCorner" name="brCorner" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="cornerIndex" stdset="0">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectorDragCorner</class>
   <extends>QWidget</extends>
   <header>ProjectorDragCorner.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
